### DISCUSSION TITLE
draft-ietf-quic-multipath version

### DISCUSSION THREAD

**Discussion by xuntyz** [Tencent/tquic#379] [General]
Which draft version does the multipath transmission protocol used in tquic?

**Comment by iyangsj** [comment]
@xuntyz It is based on the [draft-ietf-quic-multipath-05](https://datatracker.ietf.org/doc/html/draft-ietf-quic-multipath-05) but doesn't fully adopt some of its complex designs. As you can see,[ later versions of the draft](https://datatracker.ietf.org/doc/html/draft-ietf-quic-multipath) simplified some of these complex designs, making them more concise and elegant. 

To ensure interoperability, you currently need to use tquic for both the client and the server. 

Since the design of multipath may still change significantly, we will support the latest version of the draft at an appropriate time.

**Comment by polatmete** [comment]
Hi, I came across this discussion while evaluating TQUIC for a probing tool I’m building. I noticed that the multipath implementation is based on **draft-ietf-quic-multipath-05**.

Since this discussion, the QUIC multipath draft has evolved quite a bit - for instance, **[draft-ietf-quic-multipath-09](https://datatracker.ietf.org/doc/html/draft-ietf-quic-multipath-09)** introduced `initial_max_path_id` to negotiate multipath capability, and the [latest version (**-17**)](https://datatracker.ietf.org/doc/html/draft-ietf-quic-multipath-17) includes further updates and simplifications.

Are there any plans to update TQUIC’s implementation to follow a more recent draft version, or to track the current spec as it stabilises? It would be useful to know for interoperability and testing purposes.
